mono/mini/interp/interp.c

```diff
@@ -3206,6 +3207,7 @@ interp_exec_method_full (InterpFrame *frame, ThreadContext *context, FrameClause
 	GSList *finally_ips = NULL;
 	const guint16 *ip = NULL;
 	stackval *sp;
+	gboolean interp_in_gc_safe_state = FALSE;
```

Comment:
benchmarking showed that it's very beneficial to have lower stack usage. Maybe we can have this somewhere else. Like in the context.

Reply:
Heh, I was "afraid" of such a comment 😉 

`ThreadContext` wouldn't work as you can have several `managed-to-native` frames on the stack, right? Other ideas?